lib/language.js

```diff
@@ -7,42 +7,38 @@
 const locale = require('locale')
 
 const cookieName = 'language'
+const cookieOptions = { secure: true, sameSite: 'strict', httpOnly: true }
 const validLanguages = ['sv', 'en']
 const defaultLanguage = validLanguages[0]
 const supportedLocales = new locale.Locales(validLanguages, defaultLanguage)
 
+const validLanguage = lang => {
+  if (validLanguages.includes(lang)) {
+    return lang
+  }
+}
+
 /**
  * Initialize locale and language for the current user session (respects cookie: language).
  */
 function _init(req, res, newLang) {
-  let lang = newLang || req.cookies[cookieName]
-
-  // Only allow lang to be one of the valid languages
-  if (lang && !validLanguages.includes(lang)) lang = defaultLanguage
+  let lang =
```

Comment:
```suggestion
  const lang =
```

Since we no longer assign it a different value?

Reply:
Yes!